### PR TITLE
walkthrough simplification

### LINKED ISSUE #8: Document first time OSA creation error because OSA tenant is not allowed to manage subscription
Typically in ARM subs are only registered and managed by one tenant (their home tenant). However, sometimes some aspects of a subscription can be managed by managed application publisher tenant. For this subscription record needs to be updated and this happens the first time an operation happens on sub id + publisher tenant combination.

This operation can fail sometimes with the error below because it takes times for cache to be invalidated. 

>The access token for this request was issued by the tenant '<TENANT ID>' which is neither the owner tenant '<TENANT ID>' nor one of the tenants '' which can manage this subscription '<SUBSCRIPTION ID>'. The access token must be issued from proper tenant.

Add documentation that customer might need to wait 10-15mins for cache to invalidated and data to be replicated.

### PR DESCRIPTION
**Pull request by jim-minter**
fixes #8 
(among many other things)

### CODE REVIEW

File: docs/known-issues.md [hunk 1]
```diff
@@ -29,6 +29,57 @@ Get-AzureRmMarketplaceTerms -Publisher osatesting -Product open-shift-azure-prox
...
+
+## First OSA cluster creation in a subscription may fail due to access token error
+
+Currently, the first OSA cluster creation in a subscription may fail with an
+error similar to `The access token for this request was issued by the tenant
+'...' which is neither the owner tenant '...' nor one of the tenants '...' which
+can manage this subscription '...'. The access token must be issued from proper
+tenant`.
+
+If this error occurs, it is due to an ARM cache invalidation issue.  Delete the
```
Comment (amanohar) [inline docs/known-issues.md]: I don't think this text is needed here.

File: docs/openshift-on-azure-walkthrough.md [hunk 2]
```diff
@@ -8,170 +8,122 @@ ms.topic: quickstart
...
-  "location": "eastus",
-  "managedBy": null,
-  "name": "myOSACluster",
-  "properties": {
-    "provisioningState": "Succeeded"
-  },
-  "tags": null
-}
-```
+This walkthrough will create a Managed OpenShift on Azure cluster in a new
```
Comment (amanohar) [inline docs/openshift-on-azure-walkthrough.md]: Too many redundant `This walkthrough` phrases. Two of them can be removed.

File: docs/openshift-on-azure-walkthrough.md [hunk 3]
```diff
@@ -8,170 +8,122 @@ ms.topic: quickstart
...
+resource group with the same name as the cluster.
 
-## Step 3: Create an OpenShift cluster (in a default VNET)
+This walkthrough follows the `az openshift create` default behaviour of
+automatically creating an Azure AD Application for you.  For more details, see
+[AAD Application Configuration](aad-application-configuration.md).
 
-Use the `az openshift create` command to create an OpenShift cluster. 
-The following example creates a cluster named *myOSACluster* with four nodes.
+This walkthrough does not enable vnet peering on the cluster's vnet.  For
```
Comment (amanohar) [inline docs/openshift-on-azure-walkthrough.md]: `VNET`

Comment (amanohar) [inline docs/openshift-on-azure-walkthrough.md]: VNET

File: docs/vnet-peering.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+ms.service: container-service
+ms.topic: quickstart
+ms.date: 10/04/2018
+---
+
+### Vnet Peering
+
+The `az openshift create` command can create a Managed OpenShift on Azure
+cluster and peer its vnet to another vnet.  This behaviour is enabled with the
```
Comment (amanohar) [inline docs/vnet-peering.md]: `VNET`

Comment (amanohar) [inline docs/vnet-peering.md]: feature

**Comment by jim-minter** [comment]
@amanohar all fixes made, suggest a squash merge here in due course

**Pull request merged**